### PR TITLE
Add team plugin documentation

### PR DESCRIPTION
**Pull request by ivester**
Related PRs:
* https://github.com/livingdocsIO/livingdocs-editor/pull/6024
* https://github.com/livingdocsIO/livingdocs-server/pull/4974

### CODE REVIEW

File: content/reference/document/metadata/metadata-plugin-list.md [hunk 1]
```diff
@@ -1156,16 +1157,99 @@ metadata: [
 ]
 ```
 
+## li-task-list
+
+{{< added-in release-2023-03 >}}
+
+The task list meta data plugin `li-task-list` is a read only component which is used in the table dashboard. It gives an overview of current tasks connected with the given document.
```
Comment (DaRaFF) [inline content/reference/document/metadata/metadata-plugin-list.md:1164]: I don't think this is a Metadata Plugin. There is no plugin and no data are stored, it's just a representation of the `li-task-v2` Metadata Plugin.

//fya @benib what is your thought about that?

**Review by DaRaFF — APPROVED** [review]

File: content/reference/document/metadata/metadata-plugin-list.md [hunk 2]
```diff
@@ -1156,16 +1157,99 @@ metadata: [
...
+
+[See Table Dashboard Config]({{< ref "/reference/project-config/editor-settings#upstream-components" >}})
+
+## li-team
+
+{{< added-in release-2023-03 >}}
+
+The team meta data plugin `li-team` enables a user to associate other users with the current document by adding or removing them from the teams plugin. Additionally one user can be marked as the owner. The owner can not be removed from the team.
+The plugin is available in the metadata form and the table dashboard.
```
[the remark this answers is not in the document]
Comment (ivester) [reply content/reference/document/metadata/metadata-plugin-list.md:1173]: a368771d4c70e9270d87a0d64749e7a4d1b8d85a

File: content/reference/document/metadata/metadata-plugin-list.md (hunk 1 above)
Comment (ivester) [reply content/reference/document/metadata/metadata-plugin-list.md]: @benib @DaRaFF 
4d5aab84c520445c3b1dc82cae826aeb47fda411

Comment (benib) [reply content/reference/document/metadata/metadata-plugin-list.md]: This is not a metadata plugin, therefore it should not be mentioned in the metadata plugin list

**Pull request merged**